StreemNow/StreemInitializer.swift

```diff
@@ -20,14 +20,15 @@ class StreemInitializer {
     private var launchedByInviteId: String?
 
     private let appId = "*** YOUR APP-ID GOES HERE ***"
+    private let streemDomain = "sandbox.streem.cloud"
```

Comment:
Maybe add a comment at the end of the line? Something like: `// for Production, set streemDomain to nil`

Reply:
Great call!